public/keymaps/a/afternoonlabs_breeze_rev1_default.json

```
@@ -0,0 +1,29 @@
+{
+   "keyboard": "afternoonlabs/breeze/rev1", 
+   "keymap": "default", 
+   "layout": "LAYOUT", 
+   "commit": "b32b928ff3dea8a3d396ff44d491bb3d4a95a2c8",
```

This change should also go on the rev0 keymap.

```suggestion
   "commit": "64da7413a0c2b3ea555288f5d296e3a98a0da6b4",
```